### PR TITLE
{lib}[foss/2018b] preCICE v1.3.0 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/preCICE/preCICE-1.3.0_fix_bad_symlinking.patch [hunk 1]
```diff
@@ -0,0 +1,14 @@
+diff -ru precice-1.3.0.orig/CMakeLists.txt precice-1.3.0/CMakeLists.txt
```
Comment (boegel) [inline easybuild/easyconfigs/p/preCICE/preCICE-1.3.0_fix_bad_symlinking.patch]: @akesandgren missing patch description + author

Did you report this upstream?

Comment (akesandgren) [reply easybuild/easyconfigs/p/preCICE/preCICE-1.3.0_fix_bad_symlinking.patch]: he he... the checksum is correct, I just forgot to upload the patchfile after fixing the checksums for the updated patchfile.

Comment (fsimonis) [reply easybuild/easyconfigs/p/preCICE/preCICE-1.3.0_fix_bad_symlinking.patch]: Greetings from upstream! We finally removed the unfortunate symlink contraption after the 1.3.0 release. https://github.com/precice/precice/commit/0f6b0a836dc3212cfb8806fe20f06c6ad8c265c7

@akesandgren many thanks for this PR! FYI we are planning to release 1.4.0 in around 2 weeks.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
janson.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Core(TM) i7-5600U CPU @ 2.60GHz, Python 2.7.12
See https://gist.github.com/edbc1290fc675f05dceb8181a378edf8 for a full test report.

File: easybuild/easyconfigs/p/preCICE/preCICE-1.3.0_fix_bad_symlinking.patch (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/p/preCICE/preCICE-1.3.0_fix_bad_symlinking.patch]: @fsimonis :+1: :)

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in this PR)
node3112.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/07b1f56b1cd2bea40eb2016f439ab9c0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2431.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7f3f5ebf0f75724cc7348d41e1b34a9c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in this PR)
node2016.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4d7d8182cf59eb5045709b2075b9ccef for a full test report.

**Comment by akesandgren** [comment]
Looks like this one needs a bit more work on the PythonInterp searching...

**Comment by akesandgren** [comment]
Replaced by #11886

**Pull request closed without merging**